Resources/Private/Partials/FrontEndEditor/Form.html

```diff
@@ -0,0 +1,37 @@
+<html xmlns:f="http://typo3.org/ns/TYPO3/CMS/Fluid/ViewHelpers" data-namespace-typo3-fluid="true">
+    <f:variable name="labelPrefix" value="plugin.frontEndEditor"/>
+    <f:variable name="propertyLabelPrefix" value="{labelPrefix}.property"/>
+    <f:variable name="idPrefix" value="tea-editor"/>
```

Comment:
I personally am not a fan of the `f:variable` ViewHelper. I don't like to define variables within a View.
I often use the outer render ViewHelper in order to assign the additional context to the inner View. This doesn't work with `_all` but I see `_all` like `$GLOBALS` within PHP and Template, Partials, Sections as methods or functions. That's why I always try to pass the necessary context explicitly.

But that's just a comment, no need to adjust this. But feel free to comment with your own thoughts in order to exchange our different views on that.

Reply:
The rationale behind this variables is to ensure and communicate to the reader that I use the same label prefix, ID prefix, etc. in all places, and to also make those easier to change (as only one place needs to be changed instead of every occurrence).

Reply:
It sounds like it makes more sense to keep it your way then.

My approach would end in removing the assignments here and altering the render call like this:
```
<f:render partial="FrontEndEditor/Form" arguments="{
    propertyLabelPrefix: 'plugin.frontEndEditor.property',
    idPrefix: 'tea-editor'
}"/>
```
`labelPrefix` was unused, except to define the `propertyLabelPrefix`. That's why it is missing in my example.

But I'll close the comment. Just wanted to share my own thoughts.

Reply:
Hold on, then I'll inline `labelPrefix`.

(I'd like to keep the variables in the partial instead of in the `f:render` as I'm planning to reuse the partial for the "new tea" form, which would create code duplication with the prefixes again.)

Reply:
Done.